addon/components/ember-selectize.js

```diff
@@ -107,6 +107,11 @@ export default Ember.Component.extend({
   selectizeOptions: Ember.computed(function() {
     var allowCreate = get(this, 'create');
 
+    //Split the passed in plugin config into an array.
+    if (typeof this.plugins === 'string') {
+      this.plugins = this.plugins === "" ? [] : this.plugins.split(',');
```

Comment:
Ouch, arrow functions!

You don't need the return, do you?
Does this work?
`.map( item => item.trim() )`